.travis.yml

```diff
@@ -68,44 +69,15 @@ script:
         php -l $i || exit $?;
       done
     # Run unit tests to make sure functions still do what they should.
-    - vendor/bin/phpunit test/unittests/
+    - vendor/bin/phpunit --configuration test/phpunit.xml --testsuite 'LorisUnitTests for php/libraries'
 
     # Run integration tests to make sure everything didn't just
     # break
-    - vendor/bin/phpunit test/integrationtests/
+    - vendor/bin/phpunit --configuration test/phpunit.xml --testsuite 'Loris Core Integration Tests'
 
     # Run integration tests for specific modules
-    - vendor/bin/phpunit modules/brainbrowser/test
-    - vendor/bin/phpunit modules/candidate_list/test
-    - vendor/bin/phpunit modules/candidate_parameters/test
-    - vendor/bin/phpunit modules/configuration/test
-    - vendor/bin/phpunit modules/conflict_resolver/test
-    - vendor/bin/phpunit modules/create_timepoint/test
-    - vendor/bin/phpunit modules/dashboard/test
-    - vendor/bin/phpunit modules/data_integrity_flag/test
-    - vendor/bin/phpunit modules/data_team_helper/test
-    - vendor/bin/phpunit modules/datadict/test
-    - vendor/bin/phpunit modules/dicom_archive/test
-    - vendor/bin/phpunit modules/document_repository/test
-    - vendor/bin/phpunit modules/examiner/test
-    - vendor/bin/phpunit modules/final_radiological_review/test
-    - vendor/bin/phpunit modules/genomic_browser/test
-    - vendor/bin/phpunit modules/help_editor/test
-    - vendor/bin/phpunit modules/imaging_browser/test
-    - vendor/bin/phpunit modules/instrument_builder/test
-    - vendor/bin/phpunit modules/instrument_list/test
-    - vendor/bin/phpunit modules/instrument_manager/test
-    - vendor/bin/phpunit modules/mri_upload/test
-    - vendor/bin/phpunit modules/mri_violations/test
-    - vendor/bin/phpunit modules/new_profile/test
-    - vendor/bin/phpunit modules/next_stage/test
-    - vendor/bin/phpunit modules/reliability/test
-    - vendor/bin/phpunit modules/statistics/test
-    - vendor/bin/phpunit modules/survey_accounts/test
-    - vendor/bin/phpunit modules/timepoint_flag/test
-    - vendor/bin/phpunit modules/timepoint_list/test
-    - vendor/bin/phpunit modules/training/test
-    - vendor/bin/phpunit modules/user_accounts/test
+    - vendor/bin/phpunit --configuration test/phpunit.xml --testsuite 'Loris Module Integration Tests'
```

Comment:
The only problem I have with putting them all into a test suite like this is with Selenium's random failures, one of the solutions was to write a wrapper that reruns the failed integration test and only fails if the test fails both times. Is there a way to do that it they're all launched from one phpunit invocation like this?

(It's no worse than the current state of things, so I wouldn't hold up this pull request based on this, but we might need to re-split it later..)